packages/core/src/base-axis-chart.ts

```diff
@@ -362,6 +390,18 @@ export class BaseAxisChart extends BaseChart {
 				.attr("stroke-width", Configuration.scales.domain.strokeWidth);
 		}
 
+		const tickHeight = this.getLargestTickHeight(yAxisRef.selectAll(".tick"));
+		// Add y-axis title
+		if (this.innerWrap.select(".axis-label.y").nodes().length === 0 && this.options.scales.y.title) {
+			yAxisRef.append("text")
+				.attr("class", "y axis-label")
+				.attr("transform", `rotate(-90) translate(${ - (yAxisRef.node().getBBox().height / 2)},
+				${ - (tickHeight + Configuration.scales.tick.heightAddition) * 1.5})`)
+				.attr("text-align", "center")
+				.text(this.options.scales.y.title);
+			this.svg.attr("transform", `translate(${this.innerWrap.select(".axis-label.y").node().getBBox().height}, 0)`);
+		}
```

Comment:
same comments here as the other instance of this code. Also can we reuse this somehow. We can probably use the d3 `.data()` to handle this

Reply:
Sounds good, followed same convention as what's done for x axis